examples/spark/parallel/feature-processing-only.py

```diff
@@ -0,0 +1,35 @@
+from examples.spark.examples_utils import get_spark_session, get_dataset
+from sparklightautoml.pipelines.features.lgb_pipeline import SparkLGBAdvancedPipeline, SparkLGBSimpleFeatures
+from sparklightautoml.pipelines.features.linear_pipeline import SparkLinearFeatures
+from sparklightautoml.reader.base import SparkToSparkReader
+from sparklightautoml.tasks.base import SparkTask
+
+
+feature_pipelines = {
+    "linear": SparkLinearFeatures(),
+    "lgb_simple": SparkLGBSimpleFeatures(),
+    "lgb_adv": SparkLGBAdvancedPipeline()
+}
+
+
+if __name__ == "__main__":
+    spark = get_spark_session()
+
+    # settings and data
+    cv = 5
+    feat_pipe = "lgb_adv"  # linear, lgb_simple or lgb_adv
+    dataset_name = "lama_test_dataset"
+    dataset = get_dataset(dataset_name)
+    df = spark.read.csv(dataset.path, header=True)
+
+    task = SparkTask(name=dataset.task_type)
+    reader = SparkToSparkReader(task=task, cv=cv, advanced_roles=False)
+    feature_pipe = feature_pipelines.get(feat_pipe, None)
+
+    assert feature_pipe, f"Unsupported feat pipe {feat_pipe}"
```

Comment:
lets checking would be more verbose. what are we asserting with pipeline exactly: validity, emptiness, or whatever

Reply:
fixed